wagtailmenus/tests/test_menu_rendering.py

```diff
@@ -910,6 +910,246 @@ def test_footer_flat_menu_output(self):
         """
         self.assertHTMLEqual(menu_html, expected_menu_html)
 
+    @override_settings(WAGTAILMENUS_CUSTOM_URL_SMART_ACTIVE_CLASSES=True)
```

Comment:
Hi @inostia. While I appreciate the work you've gone to here, I was hoping that we could add some simpler unit-tests for just this new method in isolation (For an example, see the [`TestFindSiteForRequest`](https://github.com/wagtail/wagtail/blob/master/wagtail/core/tests/test_sites.py#L59) test case from the Wagtail codebase). Probably added as a new file (perhaps `wagtailmenus/tests/test_menu_items.py`)

These fixture-powered tests are really just to give me an indication of how these established menus are doing, and ensure they remain consistent over time. However, they are a blunt tool, and aren't a very good way to test the workings of very specific functionality. 

The current tests in this file are a bad example (I wrote them years ago).

Reply:
Great thanks. I was using the tests in #48 as an example but this makes sense.